libs/blocks/gnav/gnav-contextual-search.js

```diff
@@ -97,11 +97,16 @@ export default async function onSearchInput({ value, resultsEl, searchInputEl, a
 
   if (currentSearch === lastSearch) {
     if (!hits.length) {
-      const advancedLink = advancedSearchEl.querySelector('a');
-      const href = new URL(advancedLink.href);
-      href.searchParams.set('q', value);
-      advancedLink.href = href.toString();
-      resultsEl.replaceChildren(advancedSearchEl);
+      const emptyMessage = createTag('p', {}, 'No results');
```

Comment:
Should "no results" be a placeholder value so it can be translated?

Reply:
Updated, thanks!